styles.css

```diff
@@ -39,47 +50,638 @@ body {
+  width: 1.25em;
+  height: 1.19em;
+  margin-right: 1.25em;
+}
+
+.contact {
+  background-color: #fff;
+}
+
+#fname {
+  margin-top: 3.25rem;
+}
+
+input {
+  width: 100%;
+  margin-bottom: 1.25em;
+  padding: 0.9375em 2.375em 0.9375em 1em;
+  border-radius: 0.5em;
+  border: 1px solid #cfd8dc;
+}
+
+input::placeholder {
+  font-family: Poppins, sans-serif;
+  font-size: 1.0625em;
+  vertical-align: center;
+  text-align: left;
+  color: #172b4d;
+  font-weight: 400;
+}
+
+textarea::placeholder {
+  color: #b3bac5;
+  font-size: 0.9375em;
+  text-align: left;
+  font-weight: 400;
+}
+
+.fieldset {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  background-image: url(images/contact\ form\ shapes-mobile@2x.png);
+  border-top-left-radius: 5em;
+  background-size: contain;
+  background-position: top right;
+  background-repeat: no-repeat;
+  margin: 0;
+  border-style: none;
+  padding: 0;
+}
+
+textarea {
+  width: 100%;
+  font-family: Poppins, sans-serif;
+  padding: 0.9375em 2.375em 0.9375em 1em;
+  border: 1px solid #cfd8dc;
+  border-radius: 0.5em;
+  resize: none;
+}
+
+.contact hr {
+  background: #0d151e;
+  height: 0.3125em;
+  width: 8.375em;
+  border-radius: 0.625em;
+}
+
+.contact button {
+  width: 8.875em;
+  height: 3em;
+  font-family: Poppins, sans-serif;
+  font-style: medium;
+  line-height: 94%;
+  font-size: 1.0625em;
+  padding: 0.75em 1em;
+  border-radius: 0.5em;
+  background: #fff;
+  color: #6070ff;
+  letter-spacing: 3%;
+}
+
+.contact button:hover {
+  cursor: pointer;
+  color: #fff;
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 0.5em;
+}
+
+.contact button:active {
+  cursor: pointer;
+  color: #fff;
+  background-color: #2230d2;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 0.5em;
+}
+
+form {
+  padding: 0 1.25rem;
+}
+
+@media screen and (min-width: 375px) {
+  .desktop {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #headermenubtn {
+    display: none;
+  }
+
+  .desktop {
+    display: flex;
+  }
+
+  nav a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .desktop a {
+    text-decoration: none;
+    color: #344563;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 0.9375rem;
+    line-height: 1.25rem;
+    margin-right: 2rem;
+    display: flex;
+    align-items: center;
+  }
+
+  nav li {
+    margin-left: 2rem;
+    font-size: 0.9375rem;
+    font-style: medium;
+  }
+
+  .desktop li {
+    list-style-type: none;
+  }
+
+  .desktop a:hover {
+    cursor: pointer;
+  }
+
+  #header {
+    background-image: url(images/header-shapes\ desktop.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    flex-direction: column;
+  }
+
+  #headerarticle {
+    display: flex;
+    flex-direction: column;
+    margin-left: 260px;
+    width: 920px;
+    margin-bottom: 80px;
+  }
+
```

Comment:
- [ ] As you move from the mobile version to the desktop version kindly consider sticking to the Figma design as much as possible kindly consider fixing this section on the mobile version by adding well the background background image to make your website an exact duplicate of the given design on both screens check the images below to see the difference 
- your design 👇 
<img width="279" alt="Screen Shot 2023-06-08 at 20 04 03" src="https://github.com/BenMKT/myPortfolio/assets/60146030/a87b4633-ab2c-462d-a1c0-f281441e5d8e">

- Given design 👇 
<img width="296" alt="Screen Shot 2023-06-08 at 20 07 04" src="https://github.com/BenMKT/myPortfolio/assets/60146030/fad8c9dd-9d81-4c57-9756-9c70b54833f5">

Reply:
- Fixed header mobile background to conform to Figma design.
- Added a working link to my GitHub and social media under the author's section to give a good user experience.